CodeMaid.config

```diff
@@ -65,4 +61,4 @@
             </setting>
         </SteveCadwallader.CodeMaid.Properties.Settings>
     </userSettings>
-</configuration>
\ No newline at end of file
+</configuration>
```

Comment:
With the addition of an `.editorconfig` file, do we need this file at all any more? I'd vote to remove this all together.